app/src/main/res/values/arrays.xml

```diff
@@ -1,20 +1,22 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources xmlns:tools="http://schemas.android.com/tools" tools:ignore="MissingTranslation">
-    <string-array name="themesEntries">
+    <string-array name="themesEntries"> //not in use at the moment
```

Comment:
Oh and please `git rebase` to the latest KISS version to resolve that merge conflict:

```
git pull --rebase https://github.com/Neamar/KISS.git master
git mergetool # For running a UI to resolve 
```